libs/designsystem/src/lib/components/page/page.component.html

```diff
@@ -66,7 +58,8 @@
 
   <!--  Sticky content -->
   <ng-container *ngIf="stickyContentTemplate">
-    <div class="sticky-content-container">
+    <span #stickyContentContainerPreamble></span>
```

Comment:
Are we adding this span strictly for the purpose of observing it with the intersection observer? 
Could we do it something like the following instead of introducing a new element: https://css-tricks.com/how-to-detect-when-a-sticky-element-gets-pinned/

Reply:
Implemented the css solution.